secboot/secboot_hooks.go

```diff
@@ -194,18 +198,6 @@ func unlockVolumeUsingSealedKeyFDERevealKeyV2(sealedEncryptionKeyFile, sourceDev
 			}
 		}
 	}()
-	// ensure that the model is authorized to open the volume
-	model, err := opts.WhichModel()
-	if err != nil {
-		return res, fmt.Errorf("cannot retrieve which model to unlock for: %v", err)
-	}
-	ok, err := modChecker.IsModelAuthorized(model)
-	if err != nil {
-		return res, fmt.Errorf("cannot check if model is authorized to unlock disk: %v", err)
-	}
-	if !ok {
-		return res, fmt.Errorf("cannot unlock volume: model %s/%s not authorized", model.BrandID(), model.Model())
-	}
```

Comment:
there'a defer here that can also go away now as there's no code that can set error in the rest of the function anymore?